src/dmd/dsymbolsem.d

```diff
@@ -2894,6 +2894,8 @@ private extern(C++) final class DsymbolSemanticVisitor : Visitor
             printf("sc.parent = %s, parent = %s\n", sc.parent.toChars(), funcdecl.parent ? funcdecl.parent.toChars() : "");
             printf("type: %p, %s\n", funcdecl.type, funcdecl.type.toChars());
         }
+	if (funcdecl.fbody && (funcdecl.storage_class & STC.disable))
```

Comment:
The indentation is wrong.

Reply:
This is why the checks are failing?

Reply:
No, an example of a failing build:

```
std/datetime/systime.d(380): Deprecation: constructor `std.datetime.systime.Clock.this` cannot be annotated with `@disable` because it has a body
```

Reply:
> This is why the checks are failing?

Yes, that's may be the case: ` Error - file 'dmd/dsymbolsem.d' contains tabs at line 2897`.

There seems to be (at least) two different errors.

Reply:
But for the deprecation in the "std/datetime/systime.d(380):" this is what the issue suggested to do; "So, I suggest that we deprecate having bodies on _all_ @disabled functions and move to making it an error like it is with @disabling the init property on a struct."  [1]. And I find the idea legit.

[1] https://issues.dlang.org/show_bug.cgi?id=14854

Reply:
FYI the current `master` version of Phobos needs to compile without deprecation messages, so you would need to fix Phobos first, before introducing deprecation messages in it.

Reply:
Ok I will work on a fix there. Thank you for the information.

Reply:
I do not understand why the checks are still giving the error for the deprecation message for "
std/internal/cstring.d(188): Deprecation: generated function `std.internal.cstring.TempCStringBuffer!char.TempCStringBuffer.__aggrPostblit` cannot be annotated with `@disable` because it has a body" even though I have already change the main library and the PR were merged.

Reply:
Did you repush to your PR after the change was merged?

Reply:
In phobos?

Reply:
So. Should u change the dlang tools as well?

Reply:
All done! Should I commit again?